mesheryctl/internal/cli/root/system/check.go

```diff
@@ -599,6 +599,6 @@ func mesheryReadinessHealthCheck() (bool, error) {
 func init() {
 	checkCmd.Flags().BoolVarP(&preflight, "preflight", "", false, "Verify environment readiness to deploy Meshery")
 	checkCmd.Flags().BoolVarP(&pre, "pre", "", false, "Verify environment readiness to deploy Meshery")
-	checkCmd.Flags().BoolVarP(&componentsFlag, "components", "", false, "Check status of Meshery components")
+	checkCmd.Flags().BoolVarP(&adapter, "adapter", "", false, "Check status of Meshery adapters")
```

Comment:
@piyushsingariya, here, `meshery-perf` will be referred to as an "adapter" to the user, is that right?

Reply:
Nope, only adapters are going to be checked here. There's still a health-check pending for Operator, MeshSync, Broker. so meshery-perf will be one of those as a Meshery-Component. 

We can't health-check meshery-perf with adapters because for Meshery Server it's not an "adapter".

Reply:
It will be confusing to still use `adapter` while they are replaced with `components`. How about using `components` instead of `adapters`, and skipping the health-check for Operator, MeshSync, Broker and meshery-perf, until they are ready?

Reply:
Good point, I thought about it last day, @gyohuangxin you're right. and we can change this health-check category with "components" and health-check each of them under the same hood.